actions/class.TestRunner.php

```diff
@@ -289,6 +294,61 @@ public function index() {
         
         $this->afterAction(false);
 	}
+
+    /**
+     * Mark an item for review in the Assessment Test Session flow.
+     *
+     */
+    public function markForReview() {
+        $this->beforeAction();
+        $testSession = $this->getTestSession();
+
+        try {
+            if ($this->hasRequestParameter('position')) {
+                $itemPosition = intval($this->getRequestParameter('position'));
+            } else {
+                $itemPosition = $testSession->getRoute()->getPosition();                
+            }
+            if ($this->hasRequestParameter('flag')) {
+                $flag = $this->getRequestParameter('flag');
+                if (is_numeric($flag)) {
+                    $flag = !!(intval($flag));
+                } else {
+                    $flag = 'false' != strtolower($flag);
+                }
+            } else {
+                $flag = true;
+            }
+            taoQtiTest_helpers_TestRunnerUtils::setItemFlag($testSession, $itemPosition, $flag);
+        }
+        catch (AssessmentTestSessionException $e) {
+            $this->handleAssessmentTestSessionException($e);
+        }
+
+        $this->afterAction();
+    }
+
+    /**
+     * Jump to an item in the Assessment Test Session flow.
+     *
+     */
+    public function jumpTo() {
+        $this->beforeAction();
+        $session = $this->getTestSession();
+
+        try {
+            $session->jumpTo(intval($this->getRequestParameter('position')));
+
+            if ($session->isRunning() === true && taoQtiTest_helpers_TestRunnerUtils::isTimeout($session) === false) {
```

Comment:
This part of code was copied from other navigation methods, and I assumed this kind of comparison was needed in that case